Formula/clozure-cl.rb

```diff
@@ -0,0 +1,63 @@
+class ClozureCl < Formula
+  desc "Common Lisp implementation with a long history"
+  homepage "https://ccl.clozure.com"
+  url "https://github.com/Clozure/ccl/archive/v1.11.5.tar.gz"
+  sha256 "07c7e05c1d50ccbf1f7602a1d436b6d6da5dcda7656b89b74daa4cf833fc7929"
+  head "https://github.com/Clozure/ccl.git"
+
+  depends_on :xcode => :build
```

Comment:
Newline between depends_on and conflicts_with

Reply:
@commitay Done